config/application.rb

```diff
@@ -108,6 +108,11 @@ class Application < Rails::Application
     config.x.mastodon = config_for(:mastodon)
     config.x.translation = config_for(:translation)
 
+    if ENV.fetch('QUERY_LOG_TAGS_ENABLED', 'false') == 'true'
```

Comment:
I added this to `application.rb`, because putting it into an initializer caused the `namespaced_controller` and `action` attributes to not be inserted. `sidekiq_job_class` was correctly there.

I am not sure what is happening here, I suspect this is a load order issue. 

https://github.com/rails/rails/issues/46103 mentions a similar issue and the docs have been updated after this, but they still say that having it in an initializer should work, so I am unsure of what is happening here.

Reply:
Looks like this explicitely needs to be defined either in `application.rb` or `environment/*.rb`, due to loading order conflicts.